### DISCUSSION TITLE
Dynamic group using 'cloud_account_id'

### DISCUSSION THREAD

**Discussion by maheshn1981** [CrowdStrike/psfalcon#198] [Q&A]
Hello,
I was trying to create a new host group and add multiple assignment rule for  Cloud Account ID , how for somereason its not taking up, however for tags it works as expected.
Could you please share the correct syntax for adding the Cloud Account ID as a assignment rule

**Comment by bk-cs** [comment]
Does `service_provider_account_id` work?

```
New-FalconHostGroup -GroupType dynamic -Name my_group -AssignmentRule "service_provider_account_id:'123456789'"
```
